ipchecking/ipChecking.go

```diff
@@ -34,57 +32,53 @@ func StrToIP(iplist []string) ([]IP, error) {
 	return rlist, nil
 }
 
+func isIPv4(ip string) bool {
+	for i := 0; i < len(ip); i++ {
+		switch ip[i] {
+		case '.':
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Isn't there any better way to do this ? at least, don't use a switch for this, you can do this:
```go
if ip[i] == '.' {
    return true
}
```

Reply:
didn't find anything better than that so it'll stay like that and yeah i'll change for a if

Reply:
Can't you use `strings.Contains(ip, ".")` ?
https://golang.org/pkg/strings/#Contains